public/locales/cn/auth.json

```diff
@@ -24,12 +24,17 @@
   },
   "forget-password": {
     "title": "忘记密码",
-    "page-title": "告诉我们你的邮箱吧！",
-    "page-subtitle": "我们会一个重置密码链接到你的邮箱里",
+    "page": {
+      "title": "告诉我们你的邮箱吧！",
+      "subtitle": "我们会一个重置密码链接到你的邮箱里"
+    },
     "email": {
       "label": "你的邮箱",
       "placeholder": "邮箱",
-      "schema": "请输入你的邮箱"
+      "schema": {
+        "required": "请输入你的邮箱",
+        "valid-email": "请输入有效邮箱"
```

Comment:
Feels like this should be `invalid-email`

Reply:
this has been updated